schemas/components/schemas/Identification.yaml

```diff
@@ -33,6 +33,9 @@ properties:
   linkedId:
     type: string
     description: A customer-provided id that was sent with the request.
+  suspect:
```

Comment:
we are adding this field, previously defined and released in the update API, to the top level of an identification (similar to tag, linkedId, being customer provided).

Reply:
Can confirm it shows up in the regular get event endpoint response too